frontend/packages/db-structure/package.json

```
@@ -1,8 +1,10 @@
 {
   "name": "@liam-hq/db-structure",
-  "private": true,
-  "version": "0.0.12",
+  "version": "0.0.0-next-20250205093225",
```

I forgot to fix this...
The revert did not seem to work. Sorry for the bad manners, but I had to revert manually.

```suggestion
  "version": "0.0.12",
```